.github/bin/release-plan/release-plan.mjs

```diff
@@ -44,6 +45,13 @@ for (const workspace of workspaces) {
 
 	let changelog = (await fs.readFile(path.join(workspace.path, 'CHANGELOG.md'))).toString();
 	if (changelog.includes('Unreleased')) {
+		const canPublishPackage = await canPublish(workspace.name);
```

Comment:
Given you're rejecting errors this will halt the execution if one of them happens? You might want to `catch` these or wrap them with a try / catch

Reply:
I think we want to halt execution in case of an error.  
This would occur very early, when building the list of packages to be released.  
So there isn't any chance of stopping halfway through the release process.

If we did add a try / catch it would also do a print of the error message and `process.exit(1)` :)

I did see that the error messages were incorrect and I've fixed those.

Reply:
I do think there is some opportunity for better code here : https://github.com/csstools/postcss-plugins/issues/1068

Reply:
Ahh I see...

Just realised that the errors were showstoppers

Reply:
Indeed showstoppers, but it's a slippery slope :D
By not catching anything it quickly becomes a mess.
So thank you for pointing this out 🙇